tests/golem/envs/docker/cpu/test_integration.py

```diff
@@ -17,31 +17,31 @@
 @ci_skip
 class TestIntegration(TestCase, DatabaseFixture):
 
-    @pytest.mark.timeout(60)  # 60 sec should be well enough for this test
     @inlineCallbacks
-    def test_io(self):
-        # Set up environment
+    def setUp(self):
+        DatabaseFixture.setUp(self)
```

Comment:
Maybe its better to do a generic super setup?
```suggestion
        super().setUp()
```

Reply:
AFAIK calling `super().setUp()` would be equivalent to `TestCase.setUp()` here, not `DatabaseFixture.setUp()`. This is not what we want.

Reply:
i always thought it would do both of the parent classes, made a small test and it shows you are right